editor/src/clj/editor/pipeline/fontc.clj

```diff
@@ -175,13 +175,13 @@
   (byte-array (transduce (map :glyph-data-size) + 0 glyph-extents)))
 
 (defn check-monospaced [semi-glyphs]
-  (let [base-advance (if-let [first-glyph (first semi-glyphs)]
+  ; We can't know if it's only one glyph. And chances are it's a dynamic font
+  (if (<= (count semi-glyphs) 1)
+    false
+    (let [base-advance (if-let [first-glyph (first semi-glyphs)]
```

Comment:
At this point, you already know that there are at least 2 semi-glyphs, so you don't need `if-let`. Just use `base-advance (:advance (first semi-glyphs))`